AndroidManifest.xml

```diff
@@ -218,49 +218,53 @@
                 <action android:name="android.intent.action.MAIN" />
             </intent-filter>
         </activity>
-		
+
 		<activity android:name="org.linphone.tutorials.TutorialBuddyStatusActivity"
 		    android:theme="@style/NoTitle">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
             </intent-filter>
         </activity>
-        
+
 		<activity android:name="org.linphone.tutorials.TutorialChatRoomActivity"
 		    android:theme="@style/NoTitle">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
             </intent-filter>
         </activity>
-        
+
 		<activity android:name="org.linphone.tutorials.TutorialHelloWorldActivity"
 		    android:theme="@style/NoTitle">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
             </intent-filter>
         </activity>
-        
+
 		<activity android:name="org.linphone.tutorials.TutorialRegistrationActivity"
 		    android:theme="@style/NoTitle">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
             </intent-filter>
         </activity>
-        
+
 		<activity android:name="org.linphone.tutorials.TutorialRttSender"
 		    android:theme="@style/NoTitle">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
             </intent-filter>
         </activity>
-        
+
 		<activity android:name="org.linphone.tutorials.TutorialRttReceiver"
 		    android:theme="@style/NoTitle">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
             </intent-filter>
         </activity>
-	
-	</application>
 
-</manifest> 
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="10e2de3aa62df2eb758b9507e550ff47b3ed1d00"
```

Comment:
This is just the API key, and not the secret... but can this use an environment variable like FABRIC_API_KEY instead?